Components/Alerts/alerts.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Vee-Ka UI | Alert</title>
+    <link rel="stylesheet" href="/CSS/header.css">
+    <link rel="stylesheet" href="/CSS/sidebar.css">
+    <link rel="stylesheet" href="/CSS/pagestyle.css">
+    <link rel="stylesheet" href="alerts.css">
+</head>
+
+<body>
+    <!-- Header Starts -->
+    <div class="header">
```

Comment:
`header` HTML tag would be more semantically correct here

Reply:
Will look into it for better user experience.
Thank You